noria-server/src/controller/sql/mir/mod.rs

```diff
@@ -153,14 +153,52 @@ impl SqlToMirConverter {
             })
     }
 
+    // TODO: remove (this is just for debugging)
```

Comment:
Should this now be removed?

Reply:
Good call. Sorry about that -- I was in a rush last week and forgot to clean up things like this.